packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -18,9 +18,9 @@
 export const messages = {
   command_failure: '%s failed to run.',
   command_canceled: '%s was canceled.',
-  force_lightning_lwc_start_text: 'SFDX: Start Local Development Server',
+  force_lightning_lwc_start_text: 'SFDX: Start LWC Code Preview Server',
   force_lightning_lwc_start_not_found:
-    'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
+    'Error: This command requires a plugin. Run \'sfdx plugins:install @salesforce/lwc-dev-server\' from the terminal to install it. For more info, https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev.',
```

Comment:
Looks like it is still pointing to the beta docs. Can you get the update the link from @ekapner

Reply:
@LisandroFernandezSF - this will be the link when the content goes live in mid January:
https://developer.salesforce.com/tools/vscode/en/codepreview/lwccodepreview